data/shaders/Cccalibrator.glsl

```diff
@@ -0,0 +1,114 @@
+/*==========================================*\
+/*=======Calibrator by PavelDurov1488=======*\
+/*==========It's not ready yet...===========*\
+/*===I need to add blur, dynamic noise...===*\
+[configuration]
+
+
+[OptionRangeFloat]
+GUIName = Luminance(Y)
+OptionName = LUMINANCE
+MinValue = 0.000
+MaxValue = 3.000
+StepAmount = 0.050
+DefaultValue = 1.200
+
+[OptionRangeFloat]
+GUIName = Orange-Cyan(I)
+OptionName = ORANGECYAN
+MinValue = 0.000
+MaxValue = 3.000
+StepAmount = 0.050
+DefaultValue = 1.200
+
+[OptionRangeFloat]
+GUIName = Magenta-Green(Q)
+OptionName = MAGENTAGREEN
+MinValue = 0.000
+MaxValue = 3.000
+StepAmount = 0.050
+DefaultValue = 1.200
+
+[OptionRangeFloat]
+GUIName = Black
+OptionName = BLACK
+MinValue = 0.000
+MaxValue = 255.000
+StepAmount = 1.000
+DefaultValue = 10.000
+
+[OptionRangeFloat]
+GUIName = White
+OptionName = WHITE
+MinValue = 0.000
+MaxValue = 255.000
+StepAmount = 1.000
+DefaultValue = 240.000
+
+[OptionRangeFloat]
+GUIName = Noise
+OptionName = NOISE
+MinValue = 0.000
+MaxValue = 50.000
+StepAmount = 1.000
+DefaultValue = 10.000
+
+[OptionRangeFloat]
+GUIName = Saturation
+OptionName = SATURATION
+MinValue = 0.000
+MaxValue = 100.000
+StepAmount = 1.000
+DefaultValue = 50.000
+
+[/configuration]
+*/
+
+float pseudoNoise(vec2 co)
+{
+return fract(sin(dot(vec2(co.x+0.513,co.y+0.4124) ,vec2(12.9898,78.233))) * 43758.5453);// *fract(sin(dot(vec2(co.x+4.231,co.y+3.143) ,vec2(12.9898,78.233)*2.0)) * 43758.5453); //pseudo random number generator
+}
+
+const vec3 RGBtoY = vec3(0.299, 0.587, 0.114);

```

Comment:
You'll probably want to use `CONSTANT` here instead of `const`, otherwise it won't compile as HLSL.

For what it's worth, I'll be moving to using spirv-cross for compiling shaders in the updated system, so this won't be needed after that.

Reply:
In D3D11 it works, actually its not used...
I adapted simple-brightness shader to do RGB->YIQ->RGB conversation without this :D